alfalfa_worker/jobs/openstudio/lib/workflow/measures/alfalfa_prepare_idf/measure.rb

```diff
@@ -1,5 +1,5 @@
 # *******************************************************************************
-# OpenStudio(R), Copyright (c) 2008-2022, Alliance for Sustainable Energy, LLC.
+# OpenStudio(R), Copyright (c) 2008-2020, Alliance for Sustainable Energy, LLC.
```

Comment:
This is not a real diff. I created this new measure and deleted old unsused code. Git somehow thought that is a move and a giant edit